Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+stage "test"
+
+def testTask(imageName, imageTag) {
```

Comment:
FYI: this sort of large ugly groovy function we can 'hide away' in another file; either in another repo like I've done in my development for engine (see [here](https://github.com/docker/jenkins-pipeline-scripts/blob/master/src/com/docker/projects/Docker.groovy)) or we should also be able to keep it adjacent in this repo -- though I'll have to do some further investigation on that to be sure.